client/src/pages/auth/Login.js

```diff
@@ -1,96 +1,110 @@
-import React, { useState } from "react";
-import { Link, Redirect } from "react-router-dom";
-import { useAuth } from "./../../redux/hooks";
-import Input from "./../../components/Input";
-import Button from "./../../components/Button";
-import Icon from "./../../components/Icon";
-import Section from "./../../components/Section";
-import Breadcrumb from "./../../components/Breadcrumb";
+import React, { useState } from 'react'
```

Comment:
which tool are you using for code styling? I think you could install eslint and prettier to share the code style guidelines between the entire team.